### PR TITLE
fix: adding a unit context to scheduler configuration

### LINKED ISSUE #300: Introduce a unit test context
Introduce a _unit_ test context on top of the _integration_ context.

See also https://github.com/jenkins-x/jx/issues/6269

### PR DESCRIPTION
**Pull request by hferentschik**
fixes #300

### CODE REVIEW

File: env/templates/jx-scheduler.yaml [hunk 1]
```diff
@@ -150,3 +150,23 @@ spec:
...
+            entries:
+              - do-not-merge
+              - do-not-merge/hold
+              - do-not-merge/work-in-progress
+              - needs-ok-to-test
+              - needs-rebase
+              - needs-security-review
+      report: true
+      rerunCommand: /test unit
+      trigger: (?m)^/test images,?(\s+|$)
```
Comment (abayer) [inline env/templates/jx-scheduler.yaml]: `s/images/unit/` =)

Comment (hferentschik) [reply env/templates/jx-scheduler.yaml]: good catch, thanks

Comment (hferentschik) [reply env/templates/jx-scheduler.yaml]: done

**Pull request merged**